docusaurus/docs/core-concepts/clients/tunnelers/04-macos.md

```diff
@@ -1,3 +1,41 @@
 # MacOS
 
-The MacOS tunneler is currently in preview. Find the app in the [Mac App Store](https://apps.apple.com/us/app/ziti-tunnel/id1460484572?mt=12)
+## Installation
+
+Install the **Ziti Desktop Edge** from the [Mac App Store](https://apps.apple.com/us/app/ziti-tunnel/id1460484572?mt=12)
+
+
+:::info Note
+
+If you already have MAC Ziti Desktop already installed and you are updating it, please make sure to turn off your MAC Ziti Desktop Edge before updating it
+
+:::
+ 
+
+And after the download and installation is complete, click on **open**
+
+ 
+
+## Enrolling 
+
+Following this [guide](/docs/core-concepts/identities/creating) to create an identity and transfer the jwt file to your MAC.
+
+From the MAC Desktop Edge display, Click on **Turn Ziti On**. The user will need to click add an identity by clicking the **+** to choose the key downloaded a while ago when we started creating the endpoint.
+
+![image](/img/clients/mac-install4.png) 
+ 
+
+Find and select the downloaded jwt file and then click on **Open**
+
+![image](/img/clients/mac-install5.png) 
+
+
+You'll be brought back to the MAC desktop edge where you will click on **Enroll**

```

Comment:
> You'll be brought back to the MAC desktop edge where you will click on **Enroll**

In the left side list, the added jwt will be displayed. Finish the enrollment process by clicking the **Enroll** button.

Reply:
done